### PR TITLE
installation: bump pytest-reana

### PR DESCRIPTION
**Pull request by tiborsimko**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 history = open("CHANGES.rst").read()
 
 tests_require = [
-    "pytest-reana>=0.7.0a2,<0.8.0",
+    "pytest-reana>=0.8.0a1,<0.9.0",
```
Comment (tiborsimko) [inline setup.py:22]: CI fails on `test_calculate_hash_of_dir(sample_workflow_workspace)`:

```
        dir_hash = calculate_hash_of_dir(sample_workflow_workspace_path)
>       assert dir_hash == "8d287a3e2240b1762862d485a424363c"
E       AssertionError: assert 'd41d8cd98f00...00998ecf8427e' == '8d287a3e2240b...2d485a424363c'
E         - d41d8cd98f00b204e9800998ecf8427e
E         + 8d287a3e2240b1762862d485a424363c
```

Comment (mvidalgarcia) [reply setup.py:22]: Has someone figured out how to fix this? I'm also having it https://github.com/reanahub/reana-commons/pull/230/checks?check_run_id=1332233349#step:5:52

Comment (mvidalgarcia) [reply setup.py:22]: The culprit is the new changes introduced [in `sample_workflow_workspace`](https://github.com/reanahub/pytest-reana/commit/dbd3c611a241c92827558519687d91c9597bd2e7#diff-94f3f0e7522435047fe70876bc2377a610a6925c86e1e6d9c0c4d8174e1baf98R440-R444) that is being called [in the test](https://github.com/mvidalgarcia/reana-commons/blob/5707ba86764a88c98d422d8b39456c7a84a11076/tests/test_utils.py#L64). @diegodelemos Do you perhaps know how to solve this? I see that now all the generated hashes in the tests are the same so there might be something wrong.

Comment (diegodelemos) [reply setup.py:22]: I see.. I didn't remember about this breaking change at all, my bad... The main problem here is that we are hardcoding hashes of directories, inside `reana-commons`, which in the past we [were populating (see `shutil.copytree`)](https://github.com/reanahub/pytest-reana/commit/dbd3c611a241c92827558519687d91c9597bd2e7#diff-94f3f0e7522435047fe70876bc2377a610a6925c86e1e6d9c0c4d8174e1baf98L441) from a preconfigured file structure, and this was happening in `pytest-reana`. I think that the solution is to completely reformulate the test and assume that the workflow workspace will be empty (new behaviour of the feature).

**Review by diegodelemos — APPROVED** [review]

**Pull request merged**